source/rules/no-unsafe-takeuntil.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { Rule } from "eslint";
+import * as es from "estree";
+import { isCallExpression, isIdentifier, typecheck } from "../utils";
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      category: "RxJS",
+      description: "Disallows the application of operators after takeUntil.",
+      recommended: false
+    },
+    fixable: null,
+    messages: {
+      forbidden: "Applying operators after `takeUntil` are forbidden."
+    },
+    schema: [
```

Comment:
This rule, `no-exposed-subjects`, and `no-sharereplay` all have options and I noticed tslint has an option description but these converted rules are missing the description. I can add them if you want, but we might want to use something different as `Lint.Utils.dedent`, maybe move `common-tags` to dependencies?

Reply:
> maybe move `common-tags` to dependencies?

Yep.

I most likely copied rule metadata from someplace else that didn't have a description. If there is a description property that's specific to the options, we should fill it out.